providers/org/arxiv/harvester.py

```diff
@@ -1,7 +1,66 @@
-from share.harvest.harvester import OAIHarvester
+import logging
+import datetime
 
+from furl import furl
+from lxml import etree
 
-class ArxivHarvester(OAIHarvester):
-    rate_limit = (1, 3)
-    time_granularity = False
-    url = 'http://export.arxiv.org/oai2'
+from share import Harvester
+
+logger = logging.getLogger(__name__)
+
+
+class ArxivHarvester(Harvester):
+    namespaces = {
+        'ns0': 'http://www.w3.org/2005/Atom'
+    }
+    start_page_num = 0
```

Comment:
As `start_page_num` is going to be instance specific define it inside of the `__init__`, but remember to call `super().__init__`